librz/core/cmd_descs/cmd_cmp.yaml

```diff
@@ -0,0 +1,145 @@
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+    args:
+      - name: string
+        type: RZ_CMD_ARG_TYPE_STRING
+  - name: c1
+    summary: Compare 8-bit data at current offset with the data at <addr>
+    cname: cmd_cmp_bits
+    args:
+      - name: addr
+        type: RZ_CMD_ARG_TYPE_RZNUM
+  - name: ca
+    summary: Compare <n> bytes of data at <addr> with the data at current offset (also return in $?)
+    cname: cmd_cmp_addr
+    args:
+      - name: addr
+        type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: n
+        type: RZ_CMD_ARG_TYPE_NUM
+  - name: cb
+    summary: Compare <n> (upto 8) bytes at current offset with a number <num> (also return in $?)
+    cname: cmd_cmp_bytes
+    args:
+      - name: num
+        type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: n
+        type: RZ_CMD_ARG_TYPE_NUM
+  - name: cc
+    summary: Compare hexdump of data of block size at <addr> with the data at current offset
+    cname: cmd_cmp_hex_block
+    args:
+      - name: addr
+        type: RZ_CMD_ARG_TYPE_RZNUM
+  - name: ccc
+    summary: Show different lines between hexdump of a block of data at <addr> wth the data at current offset
+    cname: cmd_cmp_hex_diff_lines
+    args:
+      - name: addr
+        type: RZ_CMD_ARG_TYPE_RZNUM
+  - name: ccd
+    summary: Compare disassembly of block size at <addr> and at the current offset
+    cname: cmd_cmp_disasm
+    args:
+      - name: addr
+        type: RZ_CMD_ARG_TYPE_RZNUM
+        optional: true
+  - name: cf
+    summary: Compare the contents of <file> with the data at current offset
+    cname: cmd_cmp_file
+    args:
+      - name: file
+        type: RZ_CMD_ARG_TYPE_FILE
+  - name: cu
+    summary: Unified diff commands
+    subcommands:
+      - name: cu
+        summary: Compare data at <offset> with the current offset
+        cname: cmd_cmp_unified
+        args:
+          - name: offset
+            type: RZ_CMD_ARG_TYPE_RZNUM
+        details:
+          - name: Applying patches
+            entries:
+              - text: "Apply unified hex patch"
+                comment: "cu <offset> > file; wu file"
+      - name: cu1
+        summary: Compare bytes at <offset> with the current offset
+        cname: cmd_cmp_unified1
+        args:
+          - name: offset
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: cu2
+        summary: Compare words (16-bit) at <offset> with the current offset
+        cname: cmd_cmp_unified2
+        args:
+          - name: offset
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: cu4
+        summary: Compare dwords (32-bit) at <offset> with the current offset
+        cname: cmd_cmp_unified4
+        args:
+          - name: offset
+            type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: cu8
+        summary: Compare qwords (64-bit) at <offset> with the current offset
+        cname: cmd_cmp_unified8
+        args:
+          - name: offset
+            type: RZ_CMD_ARG_TYPE_RZNUM
```

Comment:
I think we should use the same structure as we do for `c`. If we remove `c1/c2/c4/c8`, then we should probably remove also `cu1/cu2/cu4/cu8` similarly.

Reply:
Sure. I'll do that.

Reply:
for `c1`/`c2`/`c4`/`c8`, it could have been possible to use the same as done for `wv` in `w?`. (or `w<1248><+->`).

Reply:
Yes, but it just felt easier and neater to use a single command for `c`.